cmd/src/debug_common.go

```diff
@@ -0,0 +1,137 @@
+package main
+
+import (
+	"archive/zip"
+	"context"
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+	"syscall"
+
+	"github.com/sourcegraph/src-cli/internal/exec"
+
+	"github.com/sourcegraph/sourcegraph/lib/errors"
+)
+
+/*
+General Stuff
+TODO: file issue on the existence of OAuth signKey which needs to be redacted
+TODO: Create getSiteConfig function
+*/
```

Comment:
Are these TODOs still relevant? If not you should remove them.

Reply:
I should file an issue for that OAuth signKey in general during testing here I encounter a number of things that should probably be REDACTED in the way code host tokens are. cc: @andreeleuterio

Reply:
I've added more descriptive TODO's here, will be sure to remove them when I'm satisfied they aren't worth having